app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -159,4 +164,22 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
         EitherT.pure[Future, String, UserInfo](UserInfo.AlreadyHasAccount(value))
 
     }
+
+  def createAccount: Action[AnyContent] = authorisedFromStride { implicit request ⇒
+    checkSession(SeeOther(routes.StrideController.getEligibilityPage().url),
+                 checkIsEligible(result ⇒ helpToSaveConnector.createAccount(result.nSIUserInfo)
+        .fold(
+          error ⇒ {
+            logger.warn(s"error during create account call, error: $error")
+            InternalServerError(views.html.technical_error())
+          }, {
+            case AccountCreated ⇒
+              Ok(routes.StrideController.getAccountCreatedPage().url)
+            case AccountAlreadyExists ⇒
+              Ok(routes.StrideController.accountAlreadyExists().url)
+          }
+        ))
+    )
+  }(routes.StrideController.getTermsAndConditionsPage().url)
```

Comment:
this is not `getTermsAndConditionsPage` page  
please fix the copy-paste error for existing endpoint `handleDetailsConfirmed` too (and may be for others too)

Reply:
and nice to add tests to test this redirectionOnLoginUrls incase user not logged in , something like below 
```
"handling detailsConfirmed" must {

      "handle the case when user is not logged in" in {
        mockAuthFail()
        val result = controller.handleDetailsConfirmed(FakeRequest())
        status(result) shouldBe SEE_OTHER
        redirectLocation(result) shouldBe Some("/stride/sign-in?successURL=%2Fhelp-to-save-stride%2Fdetails-confirmed&origin=help-to-save-stride-frontend")
      }
```

and ` mockAuthFail()` could be something like 

``` def mockAuthFail(): CallHandler4[Predicate, Retrieval[Enrolments], HeaderCarrier, ExecutionContext, Future[Enrolments]] =
    mockAuthorised(AuthProviders(PrivilegedApplication), allEnrolments)(
      Left(BearerTokenExpired("no login session exists")))
```

Reply:
Does the getEligibilityPage().url belong on the end of the handleDetailsConfirmed method? I'm not sure which page should be on the end of the createAccount method?

Reply:
test added! :)

Reply:
ask me , i will show you :)